frontend/src/app/(routes)/(overview)/overview-components/Asset.tsx

```diff
@@ -52,27 +52,42 @@ const Asset = ({
   return (
     <tr>
       <td>
-        <div>{formatCoin(asset.balance, asset.displayDenom)}</div>
-        {showChainName && (
-          <div className="text-xs text-[#a7a2b5] font-thin leading-[normal]">
-            on{' '}
-            <Link href={`/overview/${asset.chainName}`}>{asset.chainName}</Link>
+        <div className="h-[36px] flex flex-col justify-between">
+          <div className="text-sm not-italic font-normal leading-[normal] h-[14px]">
+            {formatCoin(asset.balance, asset.displayDenom)}
           </div>
-        )}
+          {showChainName && (
+            <div className="text-[10px] not-italic font-normal leading-[normal] h-[14px]">
+              on{' '}
+              <Link href={`/overview/${asset.chainName}`}>
+                {asset.chainName}
```

Comment:
I suggest displaying default info or '-' instead of null or undefined.

Reply:
{asset.chainName || '-' || 'No chain name found'}

Reply:
Not only here, wherever we are displaying the information from an API.

Reply:
I think it was already handled. This was implemented in such a way that it will never be undefined. Even typescript wont allow it.